scripts/webpack/getResolveAlias.js

```diff
@@ -4,26 +4,48 @@ const findRepoDeps = require('../monorepo/findRepoDeps');
 const findGitRoot = require('../monorepo/findGitRoot');
 const { readConfig } = require('../read-config');
 
+function getOutputPath(packageJson) {
+  for (const mainField of ['module', 'main']) {
+    const main = packageJson[mainField];
+    if (main && main.includes('dist/es')) {
+      return 'dist/es';
+    } else {
+      return 'lib';
+    }
+  }
+
+  return 'lib';
+}
+
 function getResolveAlias() {
   const gitRoot = findGitRoot();
   const deps = findRepoDeps();
+
   const alias = {};
-  const excludedPackages = ['@uifabric/api-docs'];
+  const excludedPackages = ['@uifabric/api-docs', '@uifabric/build'];
```

Comment:
Can probably also exclude infra stuff like tslint and eslint rules?

Reply:
added them